mem.go

```diff
@@ -2,12 +2,60 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
+	"os"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace string
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
+
+func ramOSInfo() (Ram, OS) {
+	v, e := mem.VirtualMemory()
```

Comment:
use the same variable for all the errors.  I typically just use err.

I understand why you did what you did, but with what i commented below, you can get rid of it.